src/ui-kit/components/Layouts/Header/Header.styled.js

```diff
@@ -0,0 +1,70 @@
+import styled from '@emotion/styled';
+import { ICONS } from '../../../icons';
+import { Typography } from '@mui/material';
+import Toogle from '../../Toogle';
+
+export const HeaderWrap = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+export const Logo = styled(ICONS.LOGO)`
+  fill: ${(p) =>
+    p.theme.palette.mode === 'light'
+      ? p.theme.palette.primary.contrastText
+      : p.theme.palette.primary.dark};
+`;
+
+export const HeaderToogle = styled(Toogle)``;
+
+export const NotificationWrap = styled.div`
+  position: relative;
+  margin: 0 24px 0 8px;
+`;
+
+export const NotificationIcon = styled(ICONS.NOTIFICATION)`
+  height: 40px;
+  width: 40px;
+  fill: ${(p) =>
+    p.theme.palette.mode === 'light'
```

Comment:
It should be just p.theme.palette.primary.main because when you change the theme, it will change to the correct color, you change to another